src/Cli/Symfony/ConsoleApplication.php

```diff
@@ -25,24 +23,20 @@ class ConsoleApplication extends Application
      */
     protected $factory;
 
-    public function setFactory(FactoryInterface $factory): void
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    public function __construct(FactoryInterface $factory, OutputInterface $output, string $name = 'TYPO3 Surf', string $version = '3.0.0-alpha')
```

Comment:
I dislike the version here. I would prefer setting this from the outside. Other opinions?

Reply:
@simonschaufi I don´t like it either. But it has nothing to do with this PR. So let us focus on the real issue.

Reply:
ok. I thought you introduced the version number here as it was in another file before.

Reply:
@sabbelasichon I refactored it slightly. Are you ok with that?